sponsor/index.html

```diff
@@ -117,6 +92,23 @@ <h2>2024 sponsors</h2>
 
 <section id="prior-sponsors">
     <h2>Prior sponsors</h2>
+    <h4>2024 Sponsors</h4>
+    <p>
+      <a href="https://how2map.com/">how2map.com</a><br/>
+      <a href="https://illustreets.com/">illustreets</a><br/>
+      <a href="https://route4me.com/">Route4Me</a><br/>
+      <a href="https://github.com/bourgesl/marlin-renderer">Marlin-renderer</a>
+    </p>
+    <div class="col cell">
+      Donations:
+      <a href="https://github.com/prushforth">Peter Rushforth</a>,
+      <a href="https://github.com/dasbaum-ch">Marco Lucarelli</a>,
+      <a href="https://github.com/groldan">Gabriel Roldan</a>,
+      <a href="https://github.com/jodygarnett">Jody Garnett</a>,
+      <a href="https://github.com/manueltimita">Manuel Timita</a>,
+      <a href="https://github.com/aaime">Andrea Aime</a>,
```

Comment:
Hum... I'm actually donating 10EUR each month to the project through Github, should count as current sponsor... or not?

Reply:
You are listed as 2025 donations 

![image](https://github.com/user-attachments/assets/6beba74b-90b4-4e0d-831d-c0a22760ab12)